generation_config.yaml

```diff
@@ -170,7 +170,8 @@ libraries:
 - api_shortname: apphub
   name_pretty: App Hub API
   product_documentation: https://cloud.google.com/app-hub/docs/overview
-  api_description: App Hub simplifies the process of building, running, and managing applications on Google Cloud.
+  api_description: App Hub simplifies the process of building, running, and managing
```

Comment:
Do we what triggers this format change?

Reply:
Yes, it's the default behavior of `ruamel.yaml`, the library used to add a new entry to the config yaml.
ruaml.yaml inherits from PyYaml. The [docs](https://pyyaml.org/wiki/PyYAMLDocumentation) explain we can use the `width` argument when `dump`ing the yaml to prevent long lines from being split

Reply:
I had enabled automerge for this PR and it got merged before I answered. I can follow up by adding a fix to prevent string formatting.

Reply:
I'm good with it as long as this format change does not mess up @JoeWang1127 's [comparator](https://github.com/googleapis/sdk-platform-java/pull/2587).